### DISCUSSION TITLE
`ui.codemirror` does not work on my hosted site but works on local

### DISCUSSION THREAD

**Discussion by amks1** [zauberzeug/nicegui#3126] [Q&A]
### Question

Hi, 

`ui.codemirror` just shows as blank on my hosted site, but works fine when testing on local machine. 

Local machine:
![image](https://github.com/zauberzeug/nicegui/assets/68153785/af14a334-d481-4c34-90f1-77e22502868a)

Hosted:
![image](https://github.com/zauberzeug/nicegui/assets/68153785/31c9f670-265f-4e30-b981-2922bf89051f)

The code is exactly the same. I'm not sufficiently proficient in Javascript to understand why, can someone help me? Thanks.

**Comment by falkoschindler** [comment]
Oh, that's interesting! Just yesterday I noticed a very similar bug #3122, which is probably related. Your report might help find the root cause. Thanks!

**Reply by falkoschindler** [reply]
We fixed the bug in #3122. Will be part of the upcoming version 1.4.27.
